CONTRIBUTING.md

```diff
@@ -26,6 +27,100 @@ The easier it is for us to reproduce it, the faster it'll be fixed!
 
 Please don't include any private/sensitive information in your issue!
 
+## Hack On Libpod
+
+This section describe to you how to start to hack on libpod.
+
+### Prepare your environment
+
+Read the [install documentation to see how to install dependencies](install.md) .
+
+During the reading of the install documentation we will as to you to:
+- install libs and tools
+- check installed versions
+- configure network
+- how to install libpod from sources but it's not our prior for hacking
```

Comment:
I'm not sure what point you were trying to make with "but it's not our prior for hacking".  I think you can just drop most of it and just go with:

how to install libpod from source.